core/src/main/kotlin/com/alecstrong/sql/psi/core/sqlite_3_35/sqlite.bnf

```diff
@@ -0,0 +1,38 @@
+{
+  // Specify the parent parser.
+  overrides="com.alecstrong.sql.psi.core.SqlParser"
+  elementTypeClass = "com.alecstrong.sql.psi.core.SqlElementType"
+
+  implements="com.alecstrong.sql.psi.core.psi.SqlCompositeElement"
+  extends="com.alecstrong.sql.psi.core.psi.SqlCompositeElementImpl"
+  psiClassPrefix = "Sqlite"
+
+  parserImports=[
+    "static com.alecstrong.sql.psi.core.psi.SqlTypes.RENAME"
+    "static com.alecstrong.sql.psi.core.psi.SqlTypes.TO"
+    "static com.alecstrong.sql.psi.core.psi.SqlTypes.COLUMN"
+    "static com.alecstrong.sql.psi.core.psi.SqlTypes.DROP"
+  ]
+}
+overrides ::= alter_table_rules
+
+alter_table_rules ::= (
+  {alter_table_add_column}
+  | {alter_table_rename_table}
+  | alter_table_rename_column
+  | alter_table_drop_column
+) {
+  extends = "com.alecstrong.sql.psi.core.psi.impl.SqlAlterTableRulesImpl"
+  implements = "com.alecstrong.sql.psi.core.psi.SqlAlterTableRules"
+  override = true
+}
+
+alter_table_rename_column ::= RENAME [ COLUMN ] {column_name} TO {column_alias} {
+  mixin = "com.alecstrong.sql.psi.core.sqlite_3_25.psi.mixins.AlterTableRenameColumnMixin"
+  pin = 1
+}
```

Comment:
I couldn't get the tests to pass without doing this. I tried everything else I could think of, so either this is the correct way, or I missed something.

Reply:
i think this is because you're overriding SqlParser, and Sqlite3.25 also overrides the same rule, so yours takes precedence. I can't actually think of how to fix this other than what you've done so all good with me